crates/iceberg-rest-server/src/catalog/tables.rs

```diff
@@ -725,11 +752,32 @@ impl<C: Catalog, A: AuthZHandler, S: SecretStore>
         futures::future::try_join_all(write_futures).await?;
 
         transaction.commit().await?;
-
+        emit_change_event(
+            serde_json::json!({"prefix": prefix.as_ref().map(|p| p.as_str().to_string())}),
+            body,
+            "commitTransaction",
+            state.v1_state.publisher,
+        )
+        .await;
         Ok(())
     }
 }
 
+async fn emit_change_event(
+    parameters: serde_json::Value,
+    body: serde_json::Value,
+    operation_id: &str,
+    publisher: impl EventPublisher,
+) {
+    publisher
+        .publish(
+            Uuid::now_v7(),
+            operation_id,
+            &serde_json::json!({"body": body, "path_parameters": parameters}),
```

Comment:
I think we should separate the "body" (as data) from the "path_parameters" (as context attributes) in Cloudevents.
The reason is that context of Cloudevents are great for routing events or building listeners on certain events (i.e. for a specific Warehouse) without the need to parse the full body.
You can see this in this [Example](https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/spec.md#example) of a json representation of a Cloudevent: Extensions are on the outer level, while data is hidden away in its own field.

Quoting the docs:
"These attributes, while descriptive of the event, are designed such that they can be serialized independent of the event data. This allows for them to be inspected at the destination without having to deserialize the event data."

Reply:
My motivation for putting the path params inside the data section was that - based on my still superficial understanding - the path parameters are necessary information to do anything useful with the request data, without these ids it's only about some tables which made me hesitate to put them in the extensions. 

> "This allows for them to be inspected at the destination without having to deserialize the event data."
Convinces me, I'll change it, it's probably very useful to have the ability to filter down events for certain tables without deserializing the data section.